src/Couchbase.Lite.Tests.Shared/TLSIdentityTest.cs

```diff
@@ -0,0 +1,331 @@
+﻿//
+//  TLSIdentityTest.cs
```

Comment:
Does this file really need to be committed in this PR if it's all just comments?